src/components/Upload.vue

```diff
@@ -72,10 +58,6 @@ export default {
   mixins: [sizeFormatter],
   inject: ['girderRest'],
   props: {
-    closeable: {
```

Comment:
We'll need to modify `App.vue` to no longer pass this prop.

Reply:
Done.